scripts/semconv/generate.sh

```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
 set -e
+set -x
```

Comment:
```suggestion
set -ex
```
Minor change to improve readability